app/test_main.py

```diff
@@ -1,3 +1,29 @@
 from app.main import get_human_age
 
-# write your code here
+
+def test_cat_and_dog_years_zero() -> None:
+    cat_age = 0
+    dog_age = 0
+    result = get_human_age(cat_age, dog_age)
+    assert result == [0, 0]
+
+
+def test_convert_years() -> None:
+    cat_age = 100
+    dog_age = 100
+    result = get_human_age(cat_age, dog_age)
+    assert result == [21, 17]
```

Comment:
The expected result `[21, 17]` for `cat_age = 100` and `dog_age = 100` seems incorrect based on the conversion rules provided in the task description. Please verify the conversion logic and adjust the expected values accordingly.